src/data/automation_i18n.ts

```diff
@@ -81,6 +81,26 @@ export const describeTrigger = (
   hass: HomeAssistant,
   entityRegistry: EntityRegistryEntry[],
   ignoreAlias = false
+) => {
+  try {
+    return tryDescribeTrigger(trigger, hass, entityRegistry, ignoreAlias);
```

Comment:
I made this a separate function so not as to avoid potentially causing big merge conflicts in the existing function, which would change the whitespace on every line.